### DISCUSSION TITLE
Download or export courses

### DISCUSSION THREAD

**Discussion by hinfinity123** [cyberjunky/python-garminconnect#183] [Q&A]
Dear cyberjunky,
great that you provided the garminconnect package. I plan to use it for some cycling date analysis. So far, I understood that I can download or export activities from Garmin Connect. Is there a possibility to download courses as well?
Best regards, hinfinity123

**Comment by cyberjunky** [comment]
I'm not sure yet, not all endpoints are known, since it's a closed API, will keep this noted.
